end2end-test-examples/spanner/src/main/java/io/grpc/spanner/TestMain.java

```diff
@@ -50,7 +50,7 @@ public static void main(String[] args) throws Exception {
     }
 
     // use sync operation to measure the latency
-    for (int i = 0; i < 50; ++i) {
+    for (int i = 0; i < 10; ++i) {
```

Comment:
10 samples won't give us true p99 and above. I guess this was used just to speed up testing, right?

Reply:
Yeah, no need to measure the latency every run. If we really want to measure the latency, I would change it to at least 5000.